frontend/front/src/pages/Developer/DevContainer.js

```diff
@@ -9,8 +9,8 @@ import { ReduxTKQuery } from "./rtkq/ReduxTKQuery";
 
 import ConfirmationTest from "./confirmModal/ConfirmationTest";
 import RtkTesting from "./rtkDat/RtkTesting";
-import Map from "./googlemap/Map";
-import MapLink from "./googlemap/MapLink";
+// import Map from "./googlemap/Map";
+// import MapLink from "./googlemap/MapLink";
```

Comment:
I had to comment this out because it wasn't compiling locally for me otherwise :sweat_smile: Let me know if you want me to undo this.